### ISSUE TITLE
Somehow my corpse got sent into nullspace because it didnt drop from the legion made with my  corpse

### ISSUE THREAD

**Issue by Kromgar** [tgstation/tgstation#18796]
Thats... all i know. I couldn't ghost from my body and they told me it was in the nullspace void.

**Comment by optimumtact** [comment]
Can you give a little more information on what you experienced?

**Comment by Kromgar** [comment]
Oh yes my screen went black and I couldn't ghost. After the legion died my corpse didn't appear i was watching from outside my body.  When i returned to my body my game froze and i was in nullspace and couldnt ghost

**Comment by optimumtact** [comment]
So the legion killed you and ate you, then later died, but your corpse was never retrieved from it's contents and when you tried to reenter your body, the screen went black?

That should be enough for people to attempt a fix, thanks.

**Comment by Kromgar** [comment]
Yep

**Comment by Durkel** [comment]
I was the miner who killed the legion, his corpse didn't appear at all, however I did notice that where he originally died was now a chasm (I had killed the legion tendril). His ore from his mining satchels were floating above the chasm marking his death spot. I think when the legion died it sent his body to above the chasm or into or something for a brief moment and never appeared where the legion died.
